src/main/resources/com/owlcyberdefense/mil-std-2045/xsd/milstd2045_message_size.dfdl.xsd

```diff
@@ -39,10 +39,9 @@
       <element name="value" type="hdr:tIntField"
                dfdl:length="20"
                dfdl:outputValueCalc="{
-                 dfdl:valueLength(/tcpHdr:message/data/content/payload, 'bytes')
+                 dfdl:valueLength(/message/data/content/payload, 'bytes')
```

Comment:
So this message here is from the no-namespace schema? Shouldn't there be an include that brings it into this naemsapce?

Reply:
No the path really is started with message where message has no namespace. 

The trick here is the use of the absolute path is kind of weird. But as this is defined in the assembly schema, it knows the path downward from the root into the components it is using, so this seems ok.